modules/300-prometheus/images/mimir/Dockerfile

```diff
@@ -0,0 +1,31 @@
+ARG BASE_DISTROLESS
+ARG BASE_GOLANG_20_ALPINE
+
+FROM $BASE_GOLANG_20_ALPINE as artifact
+
+ARG GOPROXY
+ARG SOURCE_REPO
+
+ENV GOPROXY=${GOPROXY} \
+    SOURCE_REPO=${SOURCE_REPO} \
+    CGO_ENABLED=0 \
+    GOOS=linux \
+    GOARCH=amd64
+
+RUN apk add --no-cache make git
+RUN git clone --depth 1 --branch mimir-2.10.4 ${SOURCE_REPO}/grafana/mimir.git /mimir
```

Comment:
Could you please add new entries to the `oss.yaml` file in the module directory?